strings/names/clown.txt

```diff
@@ -26,12 +26,97 @@ Pocket
 Razzle Dazzle
 Redshirt McBeat
 Ronnie Pace
-Scootaloo
+Mime

```

Comment:
This is my unironic staticname clown name; would you mind removing it from this list so that I don't get confused with other clowns?

Reply:
lol

Reply:
^kids such a loser btw

Reply:
i can guarantee you that there is more than one clown called mime in the world without this update to the .txt